pkg/repo/mysql/repo.go

```diff
@@ -0,0 +1,58 @@
+package mysql
```

Comment:
package mysql? That might conflict with imported mysql package, could be worth keeping the package name vague like for example - package db

Reply:
shouldn't conflict, as they never be imported both in the same file